features/works/work_share.feature

```diff
@@ -4,7 +4,8 @@ Feature: Share Works
 
   @javascript
   Scenario: Share a work
-    Given I have a work "Blabla"
+    Given I am logged in as "testuser1"
+      And I post the work "Blabla"
```

Comment:
Added this as the test asserts we generate an aurhor link for `testuser1`, but the fixture actually creates the work under a random user. Not sure why this was passing in CI up to now, but I got some flaky failures with it locally.

Reply:
The reason it passes Travis CI is because it's not exactly a random user:
https://github.com/otwcode/otwarchive/blob/f0a35415338b6499caf5f71d777cc7c867c3d073/features/step_definitions/user_steps.rb#L124-L128

As long as the database is empty (which is always true for Travis CI, because it creates the database from scratch, and then cleans the database after each test), it'll always use `testuser1`.

(Is the database you're using for your local tests empty? If not, the javascript tests are supposed to wipe all of the data, because [they're supposed to use truncation for the database cleaning between tests](https://github.com/otwcode/otwarchive/blob/f0a35415338b6499caf5f71d777cc7c867c3d073/features/support/env.rb#L67-L70).)

Reply:
:point_up: It's worth checking if your test database is being cleaned up correctly to avoid issues like this in the future. I won't block the PR on this, though.

Reply:
Thanks for explaining, I only ran into this the once and haven't since, but I'll keep an eye out for future flakiness.

FWIW I think specifying the user to log in as explicitly is much better when it's later required in an assertion, and to only use a random user when we don't care about the user later in the test.